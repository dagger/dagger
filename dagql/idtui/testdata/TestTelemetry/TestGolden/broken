Expected stderr:

✔ connect X.Xs
│ ✔ starting engine X.Xs
│ ✔ connecting to engine X.Xs
│ ✔ starting session X.Xs

✘ initialize X.Xs
! input: module.withSource.initialize failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1
│ ✔ resolving module ref X.Xs
│ ✘ installing module X.Xs
│ ! input: module.withSource.initialize failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1
│ │ $ ModuleSource.resolveFromCaller: ModuleSource! X.Xs CACHED
│ │ │ ✔ upload /XXX/XXX/XXX from XXXXXXXXXXX (client id: XXXXXXXXXXX, session id: XXXXXXXXXXX) (exclude: XXXXXXXXXXX) (include: XXXXXXXXXXX) X.Xs
│ │ ✘ .asModule: Module! X.Xs
│ │ ! process "go build -o /runtime ." did not complete successfully: exit code: 1
│ │ │ ✘ Module.withSource(
│ │ │ │ │ source: $ ModuleSource.resolveFromCaller: ModuleSource! X.Xs CACHED
│ │ │ │ ): Module! X.Xs
│ │ │ ! process "go build -o /runtime ." did not complete successfully: exit code: 1
│ │ │ │ ✘ Container.withExec(args: ["go", "build", "-o", "/runtime", "."]): Container! X.Xs
│ │ │ │ ┃ # dagger/broken
│ │ │ │ ┃ ./main.go:6:6: undefined: ctx
│ │ │ │ ! process "go build -o /runtime ." did not complete successfully: exit code: 1
│ │ ✘ .initialize: Module! X.Xs
│ │ ! failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1

Error logs:

✘ Container.withExec(args: ["go", "build", "-o", "/runtime", "."]): Container! X.Xs
# dagger/broken
./main.go:6:6: undefined: ctx
! process "go build -o /runtime ." did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide: export NOTHANKS=1
