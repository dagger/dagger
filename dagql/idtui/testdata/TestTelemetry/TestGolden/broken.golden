✔ connect X.Xs
│ ✔ starting engine X.Xs
│ ✔ connecting to engine X.Xs
│ ✔ starting session X.Xs

✘ initialize X.Xs
! input: module.withSource.initialize failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1
│ ✔ resolving module ref X.Xs
│ ✘ installing module X.Xs
│ ! input: module.withSource.initialize failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1
│ │ $ ModuleSource.resolveFromCaller: ModuleSource! X.Xs CACHED
│ │ ✘ .asModule: Module! X.Xs
│ │ │ ✘ Module.withSource(
│ │ │ │ │ source: $ ModuleSource.resolveFromCaller: ModuleSource! X.Xs CACHED
│ │ │ │ ): Module! X.Xs
│ │ │ │ ✘ Container.withExec(args: ["go", "build", "-o", "/runtime", "."]): Container! X.Xs
│ │ │ │ ┃ # dagger/broken
│ │ │ │ ┃ ./main.go:6:6: undefined: ctx
│ │ │ │ │ ✘ exec go build -o /runtime . X.Xs
│ │ │ │ │ ! process "go build -o /runtime ." did not complete successfully: exit code: 1
│ │ ✘ .initialize: Module! X.Xs
│ │ ! failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide: export NOTHANKS=1

Error: input: module.withSource.initialize failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -o /runtime ." did not complete successfully: exit code: 1

Stderr:
# dagger/broken
./main.go:6:6: undefined: ctx
